consul/assets/configuration/spec.yaml

```
@@ -99,6 +99,17 @@ files:
           - <SERVICE_1>
           - <SERVICE_2>
 
+    - name: named_service_tags
```

I'm not sure about the name, what about something like this?
```suggestion
    - name: include_service_tags
```